src/launcher.c

```diff
@@ -854,7 +855,8 @@ int main(int argc, char *argv[])
     return 1;
   }
 
-  run_app(MAIN_APP_NAME, NULL, fonts_path != NULL);
+  use_nbgl = fonts_path != NULL;
```

Comment:
Would be better line 847 just above 
`  if (fonts_path) {` which could be replaced by `if (use_nbgl)`

Reply:
:+1: Good catch